LoopKitTests/CarbStoreHKFilterTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CarbStoreHKFilterTests.swift
+//  LoopTests
+//
+//  Created by Rick Pasetto on 6/25/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+import XCTest
+import LoopKit
+
+class CarbStoreHKFilterTests: XCTestCase {
+    let sampleFromCurrentApp: [String : Any] = [ "endDate": Date(), "source": HKSource.default() ]
+    let sampleFromOutsideCurrentApp: [String : Any] = [ "endDate": Date(), "source": "other" ]
+
+    func testEntriesFromAllSources() {
+        checkEntries(fromCurrentAppOnly: false)
+    }
+    
+    func testEntriesFromCurrentAppOnly() {
+        checkEntries(fromCurrentAppOnly: true)
+    }
+
+    private func checkEntries(fromCurrentAppOnly: Bool, file: StaticString = #file, line: UInt = #line) {
```

Comment:
[nit] I would drop the `file` and `line` parameters. I think you want the actual `XCT*` to indicate their actual current file/line. When run, the test name itself will indicate which test failed.

Reply:
This is important to see where the test failure was in Xcode.